src/components/sidebar/helpers/generate-product-landing-nav-items.ts

```diff
@@ -49,19 +50,32 @@ export const generateProductLandingSidebarMenuItems = (
  *
  * Depends on the data located in the associated `src/data/${product.slug}.json`
  * to be loaded into the `product` object passed.
+ *
+ * Can optionally be passed menuItems, which determine sidebar content.
+ * If menuItems are not provided, falls back to a default generated set.
  */
-export const generateProductLandingSidebarNavData = (product: ProductData) => {
+export const generateProductLandingSidebarNavData = (
+  product: ProductData,
+  menuItems?: MenuItem[]
+) => {
+  const backToLinkProps = {
+    text: `${product.name} Home`,
+    href: `/${product.slug}`,
+  }
   const levelButtonProps = {
     levelUpButtonText: 'Main Menu',
     levelDownButtonText: 'Previous',
   }
-  const menuItems = generateProductLandingSidebarMenuItems(product)
+  const menuItemsWithFallback = menuItems
+    ? menuItems
+    : generateProductLandingSidebarMenuItems(product)
   const showFilterInput = false
-  const title = product.name
+  const title = 'Install'
 
   return {
+    backToLinkProps,
     levelButtonProps,
-    menuItems,
+    menuItems: menuItemsWithFallback,
```

Comment:
@zchsh just did a little QA and noticed that these changes affect the Vault and Waypoint landing pages. They get a back to link to the current page and they also have an "Install" title:

- https://dev-portal-git-zsupdate-install-sidebar-content-hashicorp.vercel.app/vault
- https://dev-portal-git-zsupdate-install-sidebar-content-hashicorp.vercel.app/waypoint

Reply:
Ah dang 🤦 Makes total sense given the function name! Thanks for flagging 💖  I'll make a separate function to generate content for the downloads views 👍